pkg/assetfetcher/http.go

```diff
@@ -175,3 +175,18 @@ func (c *NexposeAssetFetcher) newNexposeSiteAssetsRequest(siteID string, page in
 	req, _ := http.NewRequest(http.MethodGet, u.String(), http.NoBody)
 	return req
 }
+
+// CheckDependencies is aight
+func (c *NexposeAssetFetcher) CheckDependencies(ctx context.Context) error {
+	u, _ := url.Parse(c.Host.String() + "/api/3/scans")
```

Comment:
Task: I want to see us using basic "ping" endpoints wherever possible. I'll make similar comments elsewhere, especially around datastore dependencies, but for Nexpose the equivalent would probably be an HTTP OPTIONS request to the root resource at `api/3`. Could you please 1) make the endpoint configurable and 2) make the default configuration use the root API resource? We won't worry about the HTTP verb for this.

Reply:
done